torch/csrc/jit/ir.h

```diff
@@ -423,6 +444,17 @@ struct Node : public Attributes<Node> {
     return use_it;
   }
 
+  // Enforce the select invariant on inputs
+  void assertValidInput(const Node *node) const {
```

Comment:
maybe prefix it with underscore or at least mark private?

Reply:
Well, it's already private, but I can add an underscore ;)

Reply:
Actually, if I add an underscore, I'm going to want to apply this consistently to all private methods